keyserver/README.md

```diff
@@ -0,0 +1,30 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+Generate new config file with proper format
+```
+go run utils/configen/mkconfig.go
+```
+
+Generate new key pair for signing
+``` 
+go run utils/keygen/mkkey.go -vrf -signing
```

Comment:
Cool! Thanks. Haven't had time for a complete review but is it possible to already play around with the client to let it interact with the server?